ament_clang_format/package.xml

```diff
@@ -9,6 +9,11 @@
   <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
   <license>Apache License 2.0</license>
 
+  <build_depend>python3-setuptools</build_depend>
+
+  <buildtool_depend>python3</buildtool_depend>
```

Comment:
I cargo-culted this from [ament_package's package.xml](https://github.com/ament/ament_package/blame/master/package.xml) when it built and others failed.

What's really needed here in most cases is python3-dev (which is resolved by the rosdep key python3). setuptools will pull in the `python3` debian package but not python3-dev.